xwiki-platform-core/xwiki-platform-office/xwiki-platform-office-importer/src/main/resources/document-formats.js

```diff
@@ -19,196 +19,666 @@
+    "inputFamily": "TEXT",
     "storeProperties": {
-      "DRAWING": {"FilterName": "draw_flash_Export"},
-      "PRESENTATION": {"FilterName": "impress_flash_Export"}
+      "TEXT": {
+        "FilterName": "writer8_template"
+      },
+      "WEB": {
+        "FilterName": "writerweb8_writer_template"
+      }
     }
   },
+  // New one
   {
-    "name": "HTML",
-    "extensions": ["html"],
-    "mediaType": "text/html",
+    "name": "OpenDocument Text Flat XML",
+    "extensions": [
+      "fodt"
+    ],
+    "mediaType": "application/vnd.oasis.opendocument.text-flat-xml",
     "inputFamily": "TEXT",
     "storeProperties": {
-      "SPREADSHEET": {"FilterName": "HTML (StarCalc)"},
-      "PRESENTATION": {
-        "FilterName": "impress_html_Export",
-        "FilterData": {"PublishMode": 0}
-      },
-      "TEXT": {"FilterName": "HTML (StarWriter)"}
+      "TEXT": {
+        "FilterName": "OpenDocument Text Flat XML"
+      }
     }
   },
   {
-    "name": "OpenDocument Text",
-    "extensions": ["odt"],
-    "mediaType": "application/vnd.oasis.opendocument.text",
+    "name": "Microsoft Word 2007-2013 XML",
+    "extensions": [
+      "docx"
+    ],
+    "mediaType": "application/vnd.openxmlformats-officedocument.wordprocessingml.document",
     "inputFamily": "TEXT",
-    "storeProperties": {"TEXT": {"FilterName": "writerweb8_writer"}}
+    "storeProperties": {
+      "TEXT": {
+        "FilterName": "MS Word 2007 XML"
+      }
+    }
   },
+  // New one
   {
-    "name": "OpenOffice.org 1.0 Text Document",
-    "extensions": ["sxw"],
-    "mediaType": "application/vnd.sun.xml.writer",
+    "name": "Word Open XML Document Template",
+    "extensions": [
+      "dotx"
+    ],
+    "mediaType": "application/vnd.openxmlformats-officedocument.wordprocessingml.template",
     "inputFamily": "TEXT",
-    "storeProperties": {"TEXT": {"FilterName": "StarOffice XML (Writer)"}}
+    "storeProperties": {
+      "TEXT": {
+        "FilterName": "MS Word 2007 XML"
+      }
+    }
   },
   {
-    "name": "Microsoft Word",
-    "extensions": ["doc"],
+    "name": "Microsoft Word 97-2003",
+    "extensions": [
+      "doc"
+    ],
     "mediaType": "application/msword",
     "inputFamily": "TEXT",
-    "storeProperties": {"TEXT": {"FilterName": "MS Word 97"}}
+    "storeProperties": {
+      "TEXT": {
+        "FilterName": "MS Word 97"
+      }
+    }
   },
   {
-    "name": "Microsoft Word 2007 XML",
-    "extensions": ["docx"],
-    "mediaType": "application/vnd.openxmlformats-officedocument.wordprocessingml.document",
-    "inputFamily": "TEXT"
+    "name": "HTML",
+    "extensions": [
+      "html"
+    ],
+    "mediaType": "text/html",
+    "inputFamily": "TEXT", // Standard file is using "WEB"
+    "storeProperties": {
+      "SPREADSHEET": {
+        "FilterName": "HTML (StarCalc)"
+      },
+      "PRESENTATION": {
+        "FilterName": "impress_html_Export",
+        // Not among standard file
```

Comment:
The following filter is one of our custom change, I guess we don't want to lose that one.

Reply:
Same, I don't recall us making this change explicitly so I wouldn't keep the customization.

Reply:
Ok

Reply:
@mflorea so actually you added the `PublishMode` some years ago to fix https://jira.xwiki.org/browse/XWIKI-11611 I believe this is not needed anymore, but we'll probably need to double check that we can still display presentations in XWiki.

Reply:
OK, I completely forgot about this. Yes, we need to check the office preview feature (office viewer) with the standard ``document-formats.js``.

Reply:
So in the end we still need that option, we're loosing all images when exporting presentation and basically breaking the export of presentation without it.